sdk/metric/meter_test.go

```diff
@@ -1644,7 +1645,13 @@ func TestObservableExample(t *testing.T) {
 		collect := func(t *testing.T) {
 			t.Helper()
 			got := metricdata.ResourceMetrics{}
-			err := reader.Collect(context.Background(), &got)
```

Comment:
I think it would make sense to test if the values are computed correctly for different temporality.

Reply:
The test already does this.  This change just runs the test with two different readers to make sure they each get the correct metric data.

Reply:
Thanks for clarifying.